NewHorizons/Builder/Props/DetailBuilder.cs

```diff
@@ -345,8 +347,9 @@ private static void FixComponent(Component component, GameObject planetGO, bool
 
             else if (component is NomaiInterfaceOrb orb)
             {
-                orb._parentAstroObject = planetGO.GetComponent<AstroObject>();
-                orb._parentBody = planetGO.GetComponent<OWRigidbody>();
+                // detect planet gravity
+                var gravityVolume = planetGO.GetAttachedOWRigidbody().GetAttachedGravityVolume();
```

Comment:
Could this ever be null?

Reply:
GetAttachedOWRigidbody is not no, because gravity is always built before details.
GetAttachedGravityVolume is if the planet doesnt have gravity, but i check for that